packages/tailwind/README.md

```diff
@@ -3,3 +3,43 @@
 ```
 yarn add @kaizen/tailwind
 ```
+
+## Caveat before use
+
+This is an early release of the Kaizen Tailwind Preset - the API may change and evolve as we receive feedback from our teams. We will endeavour to assist where possible if classes or utilities are updated in the future.
+
+## Getting started
+
+>The assumption is that you have previously followed the FE Foundation guide for setting Tailwind up in your project.
+
+Install `@kaizen/tailwind` as a dependency and require it into your tailwind config. Then, pass it into your `presets` array.
+
+```js
+// this originally lived in @kaizen/desing-tokens but we've decoupled this for ease of versioning
+const { Preset } = require("@kaizen/tailwind")
+
+module.exports = {
+  content: ["./**/*.{ts,tsx}"],
+  presets: [Preset],
+  // important to add to the #root and #docs-root (for storybook) to ensure that tailwind classes supersede component styles
+  important: ["#root", "#docs-root"],
+  theme: {
+    extend: {},
+  },
+  plugins: [],
+}
```

Comment:
This isn't necessary right?

Alt, if you could assist with extra comments as to what each key does :)